src/test/java/GreetzLoginFavouritesTest.java

```diff
@@ -43,103 +55,79 @@ public boolean isLoggedIn(String firstName) {
+
+        Assert.assertEquals(actualItemPrice, expectedItemPrice, "Prices: ");
+        Assert.assertEquals(actualItemTitle, expectedItemTitle, "Names: ");
     }
 
-    @Test
+    @Test(priority = 5)
     public void testCardsPricingChange() throws InterruptedException {
-        driver = createDriverInstance();
-        try {
-            testLoginImpl();
 
-            driver.get("https://www.greetz.nl/kaarten/denken-aan");
-            Thread.sleep(3000);
-
-            List<WebElement> cards = driver.findElements(By.xpath("//div[@class=\"b-card-preview__container\"]"));
-            WebElement randomChosenItem = randomItem(cards);
-            randomChosenItem.click();
-            Thread.sleep(3000);
+        driver.get("https://www.greetz.nl/kaarten/denken-aan");
+        Thread.sleep(3000);
 
-            WebElement randomChosenItemQuantity = driver.findElement(By.xpath("//input[@type='number']"));
-            int chosenQuantity = getRandomNumberUsingNextInt(10) + 1;
-            randomChosenItemQuantity.clear();
-            randomChosenItemQuantity.sendKeys("" + chosenQuantity);
-            Thread.sleep(2000);
+        List<WebElement> cards = driver.findElements(By.xpath("//div[@class=\"b-card-preview__container\"]"));
+        WebElement randomChosenItem = randomItem(cards);
+        randomChosenItem.click();
+        Thread.sleep(3000);
 
-            String price = driver.findElement(By.xpath("//*[@class='price-normal']")).getText();
-            double priceOfItem = parsePriceStrToDouble(price);
+        WebElement randomChosenItemQuantity = driver.findElement(By.xpath("//input[@type='number']"));
+        int chosenQuantity = getRandomNumberUsingNextInt(10) + 1;
+        randomChosenItemQuantity.clear();
+        randomChosenItemQuantity.sendKeys("" + chosenQuantity);
+        Thread.sleep(2000);
 
-            double expectedTotalPrice = chosenQuantity * priceOfItem;
+        String price = driver.findElement(By.xpath("//*[@class='price-normal']")).getText();
+        double priceOfItem = parsePriceStrToDouble(price);
 
-            String actualTotalPriceLine = driver.findElement(By.xpath("//div[@class='price-total']")).getText();
-            double actualTotalPrice = Double.parseDouble(actualTotalPriceLine.split(" ")[2].replace(',', '.'));
-//            .substring(7, 7 + expectedTotalPriceFinal.length());
+        double expectedTotalPrice = chosenQuantity * priceOfItem;
 
-            Assert.assertEquals(actualTotalPrice, expectedTotalPrice, 0.001, "Total prices: ");
-            Thread.sleep(2000);
+        String actualTotalPriceLine = driver.findElement(By.xpath("//div[@class='price-total']")).getText();
+        double actualTotalPrice = Double.parseDouble(actualTotalPriceLine.split(" ")[2].replace(',', '.'));
 
-        } finally {
-            driver.quit();
-        }
+        Assert.assertEquals(actualTotalPrice, expectedTotalPrice, 0.001, "Total prices: ");
```

Comment:
Why actual and expected prices have a delta? :)

Reply:
Because they are a type of `double`, and I did not want to compare double on equality, considering floating points may differ at some precision.